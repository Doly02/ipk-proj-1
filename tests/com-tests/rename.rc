/auth test1 PASSWORD MyDisplayName
/rename hallo 
/join secondChannel