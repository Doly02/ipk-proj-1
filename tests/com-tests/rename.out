/auth test1 PASSWORD MyDisplayName
Success: Authentication successful.
Server: MyDisplayName joined discord.general.
/rename hallo 
/join secondChannel
Server: hallo joined secondChannel.
Success: Channel secondChannel successfully joined.